react/index.js

```diff
@@ -111,4 +111,20 @@ class Header extends Component {
   }
 }
 
-export default orderFormConsumer(injectIntl(Header))
+const orderForm = orderFormConsumer(injectIntl(Header))
```

Comment:
We should call this component to `HeaderIntl`.

Reply:
👍

Reply:
👍